code/datums/jobs/job/survivor.dm

```diff
@@ -273,20 +481,39 @@ Good luck, but do not expect to survive."})
 	wear_suit = /obj/item/clothing/suit/storage/CMB
 	shoes = /obj/item/clothing/shoes/jackboots
 	back = /obj/item/storage/backpack/satchel/sec
+	suit_store = /obj/item/storage/holster/belt/m44/full
+	belt = /obj/item/storage/belt/sparepouch
+	gloves = /obj/item/clothing/gloves/ruggedgloves
+	l_store = /obj/item/flashlight/combat
+	ears = /obj/item/radio/survivor
+	head = /obj/item/clothing/head/slouch
+
+/datum/outfit/job/survivor/marshal/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	H.equip_to_slot_or_del(new /obj/item/restraints/handcuffs, SLOT_IN_BELT)
+	H.equip_to_slot_or_del(new /obj/item/stack/medical/heal_pack/gauze, SLOT_IN_BELT)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar, SLOT_IN_BELT)
 
 
-// Rambo Survivor
+// Rambo Survivor - pretty overpowered, pls spawn with caution
 /datum/job/survivor/rambo
-	title = "Survivor"
+	title = "Overpowered Survivor"
```

Comment:
Could you just stick with "Rambo Survivor" instead

Reply:
sure